extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -151,6 +151,8 @@ export const StateProperty = {
   PREVIEW_STATE: 'previewState',
   RTL_STATE: 'rtlState',
   SHARE_MENU_STATE: 'shareMenuState',
+  // Shopping State
```

Comment:
I think this comment isn't needed. The variable name is self documenting.

Reply:
I am just following the convention of other states in the file as they have comments as well for each section, it seems like they want to be explicit. It will also help cluster related states for shopping (as we add them) together.

Reply:
I think `SHOPPING_STATE: 'shoppingState', // Shopping state` would be better then as it only refers to this one line. If you expect this list to grow then feel free to keep it, otherwise I don't think it's needed.
To be fair, the `// Story preview state` isn't needed either.

Reply:
That's a fair point. As there will certainly be more work to be done in the shopping feature (not in this quarter), I am in favor of keeping it for now for the reasons I listed above. As more shopping or shopping related states are added we can modify the comment to reflect the new changes. 

If you'd like to purge some of the other comments from this file like the Story preview state you mentioned (once again, I agree that some comments are not needed) we can open a separate PR and decide on the convention of the comments in the file.